modules/mensa/mensaCommand.ts

```diff
@@ -1,6 +1,6 @@
-import {ISlashCommand} from "../types";
```

Comment:
Rename to ...MenuListener or ...MenuEventHandler

Reply:
Assuming you meant the koeriSelectionMenu, closing this